README.rst

```diff
@@ -295,26 +295,28 @@ An optional config file may be supplied and may include:
 
 Per Project Command Line Args
 -----------------------------
-Projects may include a `.bandit` file that specifies command line arguments
+Projects may include a `.bandit.yml` file that specifies command line arguments
```

Comment:
You have `.bandit.yml` now

Reply:
my bad, I will make sure that every file mentions the same config filename